libs/domain/content/article/article.component.ts

```diff
@@ -26,27 +26,32 @@ export class ArticleComponent extends ContentMixin<ContentArticleOptions>(
   protected contentService = resolve(ContentService);
   protected contextController = new ContextController(this);
 
-  protected $articleContext = signal(
+  protected $articleId = signal(
     this.contextController.get<string>(ArticleContext.Id)
   );
+  protected $articleType = signal(
+    this.contextController.get<string>(ArticleContext.Type)
+  );
 
   protected $data = computed(() => {
-    const article = this.$articleContext();
+    const { entities } = this.$options();
+    const id = this.$articleId();
+    const type = this.$articleType();
 
-    return article
-      ? this.contentService.get({ article, entities: this.$options().entities })
+    return id && type
+      ? this.contentService.get({ id, type, entities })
       : of(null);
   });
 
   protected override render(): TemplateResult | void {
     const data = this.$data();
 
-    if (!data?.article) {
+    if (!data?.content) {
       return;
     }
 
     return html`<oryx-text
-      .content=${marked.parse(data.article)}
+      .content=${marked.parse(data.content)}
```

Comment:
wondering if we should not move markdown conversion to the adapter layer, since we do not know which element comes with which type of content.

Reply:
As it's temporary implementation I thought it's ok if component handle it. OFC in real world we should do it in normalizers

Reply:
What do you think if we move this component into labs domain too?